src/main/java/ch/njol/skript/entity/BoatChestData.java

```diff
@@ -1,54 +1,43 @@
 package ch.njol.skript.entity;
 
 import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemData;
 import ch.njol.skript.aliases.ItemType;
 import ch.njol.skript.lang.Literal;
 import ch.njol.skript.lang.SkriptParser;
 import org.bukkit.Material;
 import org.bukkit.entity.Boat;
 import org.bukkit.entity.ChestBoat;
-import org.bukkit.entity.boat.*;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.EnumMap;
+import java.util.HashMap;
 import java.util.Locale;
+import java.util.Map;
 import java.util.Random;
 
+// For <1.21.3 compatability only. 1.21.3+ boats are SimpleEntityDatas
 public class BoatChestData extends EntityData<ChestBoat> {
 
 	private static final boolean IS_RUNNING_1_21_3 = Skript.isRunningMinecraft(1, 21, 3);
```

Comment:
```suggestion
	private static final boolean IS_RUNNING_1_21_3 = Skript.isRunningMinecraft(1, 21, 2);
```
one more... just for good luck lets make it 3